### PR TITLE
Add basic logging

### PR DESCRIPTION
**Pull request by mmurrell-r7**
This PR implements some basic logging in the Main process.

By default, logs are written to the following locations:
> on Linux: ~/.config/Awsaml/logs/main.log
> on macOS: ~/Library/Logs/Awsaml/main.log
> on Windows: %USERPROFILE%\AppData\Roaming\Awsaml\logs\main.log

### CODE REVIEW

File: src/main/index.js [hunk 1]
```diff
@@ -76,7 +79,7 @@ app.on('ready', async () => {
   const port = Server.get('port');
 
   Server.listen(port, host, () => {
-    console.log(`Server listening on ${host}:${port}`); // eslint-disable-line no-console
+    log.info(`Server listening on ${host}:${port}`); // eslint-disable-line no-console
```
Comment (dgreene-r7) [inline src/main/index.js:82]: You can remove the eslint disabling comment now.

File: src/main/index.js [hunk 2]
```diff
@@ -95,6 +98,7 @@ app.on('ready', async () => {
   });
 
   mainWindow.on('close', () => {
+    log.info('BrowserWindow.webContents.close');
```
Comment (dgreene-r7) [inline src/main/index.js:101]: Can we add a `[Event]` prefix for logs we're using to debug event handling?

Comment (dgreene-r7) [reply src/main/index.js:101]: Also, to be slightly pedantic, these are events on `BrowserWindow` not `BrowserWindow.webContents`

**Review by dgreene-r7 — COMMENTED** [review]
Some tiny nits. Also just wanted to make sure that there aren't any events that we're not covering. Do we know what electron's BrowserWindow startup process looks like in terms of event dispatch?

File: src/main/index.js (hunk 2 above)
Comment (mmurrell-r7) [reply src/main/index.js]: Added the `[Event]` prefix and changed the event message to be a bit more accurate.

File: src/main/index.js [hunk 3]
```diff
@@ -111,27 +115,34 @@ app.on('ready', async () => {
...
-
   mainWindow.on('ready-to-show', () => {
+    log.info('[Event] BrowserWindow ready-to-show');
     mainWindow.show();
   });
 
-  mainWindow.webContents.on('did-finish-load', () => loadTouchBar(mainWindow, storedMetadataUrls));
+  log.info('BrowserWindow.loadURL');
+  await mainWindow.loadURL(baseUrl);
+
```
Comment (dgreene-r7) [inline src/main/index.js:137]: Can you add a log line here? Something like `log.info('BrowserWindow.loadURL completed')`?

Comment (mmurrell-r7) [reply src/main/index.js:137]: Great idea! Done.

**Review by dgreene-r7 — APPROVED** [review]

**Pull request merged**